docs/topics/spiders.rst

```diff
@@ -181,9 +181,9 @@ scrapy.Spider
        scraped data and/or more URLs to follow. Other Requests callbacks have
        the same requirements as the :class:`Spider` class.
 
-       This method, as well as any other Request callback, must return an
-       iterable of :class:`~scrapy.Request` and/or :ref:`item objects
-       <topics-items>`.
+       This method, as well as any other Request callback, must return a
+       :class:`~scrapy.Request` object, an :ref:`item object <topics-items>`, or
+       an iterable of these objects. 
```

Comment:
I think with the current proposal someone could understand that the iterable is an iterable of 1 request and 1 item.

I would rather be verbose than leave room for interpretation, specially since I believe this is not the first time someone ask for clarification here:

```suggestion
       an iterable of :class:`~scrapy.Request` and/or :ref:`item 
       <topics-items>` objects.
```

Reply:
Okay. I have changed the wording to be more verbose.